core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -152,6 +190,24 @@ private static void assertPathDeleted(GenericContainer<?> container, String path
         assertThat(actualResult.getExitCode()).isEqualTo(0);
     }
 
+    private static Map<String, FileInfo> getRpmFiles(String lines)
+    {
+        Splitter splitter = Splitter.onPattern("\s+").trimResults();
+        ImmutableMap.Builder<String, FileInfo> builder = ImmutableMap.builder();
+        for (String line : Splitter.on("\n").split(lines.trim())) {
+            List<String> columns = splitter.splitToList(line);
+            boolean isLink = columns.get(0).startsWith("l");
+
+            builder.put(columns.get(8), new FileInfo(
```

Comment:
nit: assert the number of items in `columns`, document the expected output of `rpm -qlpv`, and use named variables. Maybe something like:
```
// expect lines to look like:
// drwxr-xr-x    2 root    root     0 Apr  7  2021 /some/file
String permissions = columns.get(0);
String owner = columns.get(2);
String group = columns.get(3);
String path = columns.get(8);
String link = columns.get(10);
new FileInfo(path, PosixFilePermissions.fromString(permissions.substring(1)), owner, group, link);
```

or do that in an alternative `FileInfo` constructor.

Reply:
Yeah. I'll actually create more versatile assertions for these

Reply:
Did you push any changes here? It looks the same.

Reply:
Not yet

Reply:
it is now, ptal